packages/graphql-language-service-server/src/findGraphQLTags.ts

```diff
@@ -254,8 +254,7 @@ export function findGraphQLTags(
       }
     },
     TemplateLiteral(node: TemplateLiteral) {
-      const hasGraphQLPrefix =
-        node.quasis[0].value.raw.startsWith('#graphql\n');
+      const hasGraphQLPrefix = node.quasis[0].value.raw.startsWith('#graphql');
```

Comment:
This makes the matching similar to the [syntax highlight here](https://github.com/graphql/graphiql/blob/57b5fcd8749bd1c54f4a6a04a9ab07316fd05e71/packages/vscode-graphql-syntax/grammars/graphql.js.json#L75), allowing `#graphql:suffix`.